values/strings.xml

```diff
@@ -2261,7 +2269,7 @@ By connecting the game to a TheoTown Forum account you can participate in commen
     <string name="dialog_uploadregionaccount_title">Account needed</string>
     <string name="dialog_uploadregionaccount_text">In order to upload a region you have to connect TheoTown with a TheoTown account.</string>
     <string name="dialog_uploadregion_title">Upload region</string>
-    <string name="dialog_uploadregion_text">This will copy this region to the public list of online regions.\n\nOnce the region is uploaded you can claim parts on it online. If no password is set other players can join there then, too.</string>
+    <string name="dialog_uploadregion_text">This will copy this region to the public list of online regions.\n\nOnce the region is uploaded you can claim parts on it online. If no password is set other players can join there then, too.\n\nBy doing this, you agree that your region follows the rules which you can read here: https://forum.theotown.com/viewtopic.php?p=160325#p160325</string>
```

Comment:
@LobbyDivinus this should be improved by adding a link to the rules instead of it as a raw string.

Reply:
Yes, there will be a Rules button that will open that page.